the-american-midland-naturalist.csl

```diff
@@ -0,0 +1,175 @@
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+        <text variable="year-suffix"/>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="article-journal article-magazine" match="any">
+        <group delimiter=", ">
+          <text variable="container-title" font-style="italic" form="short"/>
+          <group delimiter=":">
+            <text variable="volume" font-weight="bold"/>
+            <text variable="page" form="short"/>
+          </group>
+        </group>
+        <choose>
+          <if variable="page volume" match="none">
+            <text variable="DOI" prefix=", doi: "/>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="chapter-info">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <group delimiter=". ">
+          <group delimiter=" ">
+            <label plural="never" variable="page" form="short"/>
+            <text variable="page"/>
+          </group>
+          <group delimiter=" ">
+            <text term="in" text-case="capitalize-first" font-style="italic" suffix=":"/>
+            <text macro="editor-translator" suffix="."/>
+            <text variable="container-title"/>
+          </group>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=", ">
+      <text variable="publisher" strip-periods="false"/>
+      <text variable="publisher-place"/>
+    </group>
+  </macro>
+  <macro name="book-access">
+    <choose>
+      <if type="book" match="any">
+        <group delimiter=" ">
+          <text variable="number-of-pages"/>
+          <label variable="page" form="short" plural="never"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year-suffix">
+    <sort>
+      <key macro="year-date"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=", ">
+        <text macro="author-short"/>
+        <text macro="year-date"/>
+        <group>
+          <label variable="locator" form="short"/>
+          <text variable="locator" prefix=" "/>
+        </group>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" subsequent-author-substitute="---" subsequent-author-substitute-rule="complete-all">
```

Comment:
> Can somebody tell me if I did the author substitution with --- as in the example?

The example PDF you linked to, http://www.emwh.org/pdf/elk/Changes%20in%20Elk%20Distribution%20and%20Group%20Sizes%20After%20Wolf%20Restoration%202012.pdf, has this:

<img width="614" alt="image" src="https://cloud.githubusercontent.com/assets/77951/26711195/b5e42d04-472c-11e7-9150-62c67236a345.png">

This means you need "partial-each" instead of `subsequent-author-substitute-rule="complete-all"`.

(see http://docs.citationstyles.org/en/stable/specification.html#reference-grouping)